<?xml version="1.0"?>
<launch>
<rosparam file="$(find eurobot_camera)/configs/camera.yaml" command="load"/> 

<node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="pixel_format" value="yuyv" />
    <param name="image_width" value="2448" />
    <param name="image_height" value="2048" />
    <param name="camera_frame_id" value="head_camera"/>
    <param name="camera_info_url" value="package://eurobot_camera/configs/calibration.yaml"/>
</node>

<node pkg="master_discovery_fkie" type="master_discovery" name="master_discovery"/>
<node pkg="master_sync_fkie" type="master_sync" name="master_sync"/>

<node name="camera_node" pkg="eurobot_camera" type="undistort.py" output="screen" args="-c /home/nuka-cola/catkin_ws/src/ros-eurobot-2019/eurobot_camera/configs/calibration.yaml -t /home/nuka-cola/catkin_ws/src/ros-eurobot-2019/eurobot_camera/resources/field_white.png">
</node>

<node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" output="screen" >
</node>
</launch>
